claims_test.go

```diff
@@ -0,0 +1,64 @@
+package jwt_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go/test"
+)
+
+const (
+	expireAtInt = 100
+	nowInt = 200
+)
+
+var claimsTestData = []struct {
+	name   string
+	need   string
+	claims jwt.Claims
+}{
+	{
+		name:   "Map claims expired json.Number",
+		need:   "Given the need to test validating a MapClaims past its json.Number expire-at time.",
```

Comment:
The `need` field is only for verbose information about the test, it isn't compared to anything. When you run `go test -v .` this plays it informs the reader the need for the test, followed by the declaration of action and then expectation. This style of writing tests is described in Go In Action.
Perhaps you mean why test the message string at all? I'd like to make sure the desired information comes through.